pgen-native/python.c

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
everything in this file is a placeholder, for now. Everything subject to change or removal, since these are just stubs.